migrations/sqlite/2019-03-05-082846_add_fqn/down.sql

```diff
@@ -1,3 +1,46 @@
 -- This file should undo anything in `up.sql`
-ALTER TABLE blogs DROP COLUMN fqn;
-ALTER TABLE users DROP COLUMN fqn;
\ No newline at end of file
+CREATE TABLE blogs_no_fqn (
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    actor_id VARCHAR NOT NULL,
+    title VARCHAR NOT NULL,
+    summary TEXT NOT NULL DEFAULT '',
+    outbox_url VARCHAR NOT NULL UNIQUE,
+    inbox_url VARCHAR NOT NULL UNIQUE,
+    instance_id INTEGER REFERENCES instances(id) ON DELETE CASCADE NOT NULL,
+    creation_date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    ap_url text not null default '' UNIQUE,
+    private_key TEXT,
+    public_key TEXT NOT NULL DEFAULT '', fqn TEXT NOT NULL DEFAULT '',
```

Comment:
`blogs_no_fqn`, `fqn TEXT NOT NULL`...

Reply:
Ah, that's why it was telling me I was not supplying enough values when doing `INSERT INTO blogs_no_fqn SELECT all, the, columns, but, not, fqn FROM blogs;`. :grimacing: